bc_obps/reporting/tests/utils/baker_recipes.py

```diff
@@ -144,6 +145,17 @@ def json_seq(json_key="generated_json", json_value="test json value", seq_value:
     emission_category=foreign_key(emission_category),
     gas_type=[foreign_key(gas_type)],
 )
+
+
+report_new_entrant = Recipe(
+    ReportNewEntrant,
+    report_version=foreign_key(report_version),
+    authorization_date=(datetime.now() - timedelta(days=10)).isoformat(),
+    first_shipment_date=(datetime.now() - timedelta(days=5)).isoformat(),
+    new_entrant_period_start=(datetime.now() - timedelta(days=20)).isoformat(),
```

Comment:
Tests should be run with a controlled set of inputs, using `now()` for a date might have unintended side-effects. I suggest hardcoding the dates in the recipe, and if tests need to change them, they can be overwritten that way